src/js/angular/core/templates/rdfResourceSearchTemplate.html

```diff
@@ -0,0 +1,25 @@
+<link href="css/rdf-resource-search.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+
+<div ng-if="getActiveRepository() && !isLoadingLocation() && hasActiveLocation()" ng-cloak>
+  <div class="card search_inputRDF" id="search_inputRDF">
```

Comment:
Do you need this ID? Also you better change the underscore with a dash in `search_inputRDF` which you put as a class.

Reply:
No, I used it at first. I will remove the ID and change the name of the class.